python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts

```diff
@@ -65,15 +72,12 @@ export const HuggingFaceAutomaticSpeechRecognitionPromptSchema: PromptSchema = {
       return_timestamps: {
         type: "string",
         enum: ["word", "char", "True", ""],
-        description: `Only available for pure CTC models (Wav2Vec2, HuBERT, etc) and the Whisper model. Not available for other sequence-to-sequence models.`
+        description: `Only available for pure CTC models (Wav2Vec2, HuBERT, etc) and the Whisper model. Not available for other sequence-to-sequence models.`,
       },
       max_new_tokens: {
-        type: "number",
-        description: `The maximum numbers of tokens to generate, ignoring the number of tokens in the prompt`
-      }
+        type: "integer",
```

Comment:
Why this change?

Reply:
Because number is a float while integer is an integer value. I think this was incorrectly transcribed the first time